xml/cl.xml

```diff
@@ -1633,7 +1638,14 @@ server's OpenCL/api-docs repository.
             <unused start="0x4200" end="0x420F"/>
     </enums>
 
-    <enums start="0x420F" end="0x4FFF" name="enums.420F" comment="Reserved for vendor extensions. Allocate in groups of 16.">
+    <enums start="0x4210" end="0x421F" name="enums.4210" vendor="Intel">
+        <enum value="0x4210"        name="CL_KERNEL_ARG_HOST_ACCESSIBLE_PIPE_INTEL"/>
+        <enum value="0x4211"        name="CL_DEVICE_MAX_HOST_READ_PIPES_INTEL"/>
+        <enum value="0x4212"        name="CL_DEVICE_MAX_HOST_WRITE_PIPES_INTEL"/>
+            <unused start="0x4213" end="0x421F"/>
+    </enums>
+
+    <enums start="0x4220" end="0x4FFF" name="enums.420F" comment="Reserved for vendor extensions. Allocate in groups of 16.">
```

Comment:
Similarly to above, this would be `name="enums.4220"`.

Reply:
Thanks.  Fixed.